index.ts

```diff
@@ -0,0 +1,5 @@
+export { Box } from './App/components/box'
+export { Button } from './App/components/Button'
```

Comment:
Is button a working component or just an example?

Reply:
Why removed the bold and font-family props in Typography?

Reply:
It's still just an example, should spend some time to make it usable, probably in this sprint)